templates/ssmtp.conf.j2

```diff
@@ -1,7 +1,20 @@
 mailhub={{ ssmtp.config.host }}:{{ ssmtp.config.port }}
 
+{% if ssmtp.config.root is defined %}
```

Comment:
Can you define these settings (ssmtp.config.root, ssmtp.config.rewritedomain and the like) but set them to false in defaults? I prefer to have all possible values defined in defaults so it's obvious what options are available.

Reply:
Should be okey now